### PR TITLE
Excel col labels

### PR DESCRIPTION
**Pull request by Snotpus**

### CODE REVIEW

**Comment by Mikuana** [comment]
Unfortunately, my use of the squash and merge has made conflicts. I would recommend merging the latest version of `Snotpus/custom_error_msg` into this branch, as that will be easier to triage commits. Once you've done that, then you can merge `dev` into this branch, and just accept all the changes from `dev` (as they'll all be effectively the same as `Snotpus/custom_error_msg`

**Comment by Snotpus** [comment]
I _think_ this is roughly where I want it... but I had a hell of a time trying to get anything more meaningful feeling out of any testing of this. I spun my wheels for a time trying to figure out how to add a private method to `_BaseSubject` which would simply list off the error messages... I think my issue may be in not understanding how to properly pass a param I'm trying to specify at the Layout-level to be associated to all of the fields within the Layout's definition

File: tests/test_other.py [hunk 1]
```diff
@@ -339,3 +339,13 @@ def test_complex_bad(bad_complex_file):
...
         assert all(x in str(ex) for x in expected_ex)
+
+
+@pytest.mark.parametrize('value,expected_output', [
+    ('1', 'A'),
+    ('1234', 'AUL'),
+    ('16384', 'XFD')
+])
+def test_excel_cell_formatter(value, expected_output):
+    print(ex.convert_to_excel_col_labels(value))
```
Comment (Mikuana) [inline tests/test_other.py:350]: No print!

**Review by Mikuana — CHANGES_REQUESTED** [review]
- Remove print statements
- Add new test file for errors, and test CellError for new Excel formatting behavior

**Pull request merged**